openfisca_us/parameters/usda/snap/income/deductions/earned_income.yaml

```diff
@@ -0,0 +1,10 @@
+description: Share of earned income that can be deducted from gross income for SNAP
+values:
+  2005-01-01: 0.2
+metadata:
+  unit: /1
+  name: snap_earned_income_deduction
```

Comment:
Thanks - we'll just need to change this on the next update to PolicyEngine (won't break it as we're hard fixing the dependency)

Reply:
Yeah it'll need a few changes, e.g. income by source instead of just market income. So far I've implemented `employment_income`, `self_employment_income`, `dividend_income`, and `interest_income`, all at the person level, though might add more to cover the legislation.